.eslintignore

```diff
@@ -1,7 +1,6 @@
-lib/internal/v8_prof_polyfill.js
```

Comment:
These files are copied directly from the v8 source, no? If so, we probably don't want to apply our coding style standards to their code?